tests/Basic/ArtworkTest.php

```diff
@@ -27,4 +29,83 @@ protected function tearDown(): void
 
         parent::tearDown();
     }
+
+    /** @test */
+    public function it_fetches_the_gallery_for_an_artwork(): void
+    {
+        $gallery = $this->make(Gallery::class, ['is_closed' => false]);
+        $galleryKey = $gallery->getAttributeValue($gallery->getKeyName());
+
+        $artwork = $this->make(Artwork::class, ['gallery_id' => $galleryKey, 'is_on_view' => true]);
+        $artworkKey = $artwork->getAttributeValue($artwork->getKeyName());
+
+        $response = $this->getJson('api/v1/artworks/' . $artworkKey);
+        $response->assertSuccessful();
+
+        $resource = $response->json()['data'];
+        $this->assertTrue($resource['is_on_view']);
+
+        Artwork::query()->delete();
+        Gallery::query()->delete();
```

Comment:
I recommend using the testing helper trait `RefereshDatabase` instead of running a bunch of delete queries:
https://laravel.com/docs/10.x/database-testing#resetting-the-database-after-each-test